lib/pbench/agent/tool_meister_start.py

```diff
@@ -1267,13 +1267,14 @@ def start(_prog: str, cli_params: Namespace) -> int:
                 "Tool installation check failures encountered",
             )
 
-        # Setup a Client API object using our existing to_client_chan object to
-        # drive the following client operations ("sysinfo" [optional] and "init"
-        # [required]).
+        # Setup a Client API object using our existing Redis server connection
+        # to drive the following client operations: "sysinfo" [optional], and
+        # "init" [required].
+        #
+        # FIXME - What should be done with the `to_client_chan` object?
```

Comment:
Um, we're not supposed to accept code with `FIXME`'s in it...right?

Reply:
Mea culpa.  That was a comment left in the PR that was merged into this PR.  I think the answer is that we should close that channel.